tools/profiler/visualize_layerwise_profile.py

```diff
@@ -151,16 +151,18 @@ def is_quant(op_name: str):
            "scaled_int8_quant" in op_name:
             return True
 
+    def is_cutlass_gemm_op(op_name: str):
+        return "void cutlass::Kernel" in op_name or \
+           "void cutlass::device_kernel" in op_name
```

Comment:
maybe we should check for `gemm` in the name too? not that we use cutlass for anything else right now, but might prevent future confusion if we use cutlass for convolution or something

Reply:
Yeah, you are right. Ill add it.